README.md

```diff
@@ -0,0 +1,43 @@
+# lapce-volt
+
+This repository serves as a starting point for writing [Lapce](https://lapce.dev/) plugins in the [Rust Programming Language](https://www.rust-lang.org).
+If you haven't used Rust before, consider familiarizing yourself with the documentation available on .
+
+## Getting started
+
+- If not yet done, [install Rust](https://www.rust-lang.org/learn/get-started)
+- This template uses `cargo-make` as a build system. Install it, too.
+- Generate your own repository based on this one by following the instructions [here](https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template).
+- [Clone it](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) locally
+- Compile your addon by executing `cargo make dev` in the cloned directory. This will create a `bin/lapce-plugin-template.wasm` file.
```

Comment:
```suggestion
- Ensure the wasi target is installed, using: `rustup target add wasm32-wasi`
- Compile your addon by executing `cargo make dev` in the cloned directory. This will create a `bin/lapce-plugin-template.wasm` file.
```

This target must be installed to compile.